### PR TITLE
Blockchain follower

### PR DESCRIPTION
**Pull request by jnordberg**

### CODE REVIEW

**Comment by GarethNelson** [comment]
General point: I know I said to keep track of other branches so merging later doesn't become a big issue, but how often should we be merging incomplete work to master?

@john-g-g @sneak your input is requested here

File: yo/blockchain_follower.py [hunk 1]
```diff
@@ -1,82 +1,170 @@
...
+
 logger = logging.getLogger(__name__)
 
 # TODO - use reliable stream when merged into steem-python
 
 # Basically this service just follows the blockchain and inserts into the DB then triggers the notification sender to send the actual notification
 
+# NOTIFICATION TYPES
+ACCOUNT_UPDATE = 'account_update'
+# Not a blockchain event ANNOUNCEMENT_IMPORTANT = 'announcement_important'
```
Comment (jnordberg) [inline yo/blockchain_follower.py]: This could be a blockchain event if we listen for a `custom_json` broadcasted by e.g. `@steemit`

Comment (GarethNelson) [reply yo/blockchain_follower.py]: I think that particular notification type is included for future uses only - idea being we can inject marketing/announcement notifications via the API and not just from the blockchain

**Comment by jnordberg** [comment]
Not implemented yet:

 * `ACCOUNT_UPDATE`: not sure what we are looking for here, just key updates or any update?
 * `REWARD`: We could listen for the `claim_reward_balance` operations but that would be pretty pointless since the user will just get a notification when clicking the claim rewards button. There's virtual `comment_reward` and `curation_reward` operations but not sure how to listen for those.
 * `FEED`: np-hard?

**Comment by GarethNelson** [comment]
account update: it's simpler to make it any change

reward: it's the virtual ops we want to listen for, I believe steemd dynamically calculates them so we may have to reimplement that logic in yo

feed: why are you saying this is np-hard? we just need to generate a feed event for each user following whenever a toplevel post is made

**Comment by john-g-g** [comment]
@GarethNelson regarding incomplete feature branches merged to master: your instinct here is correct, but until we are closer to being production ready, and as long as the branch doesn't break anything, this is ok. We're moving fast  and I want updates propagates to all yo devs asap  

account_update @jnordberg I agree with @GarethNelson "make it any change" for mvp
reward: skip for mvp
feed: probably our most valuable non-security feature. not an mvp skip, but we will add this after yo is otherwise up and running

**Pull request merged**